account_invoice_comment_template/models/account_move.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2014 Guewen Baconnier (Camptocamp SA)
+# Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2019 Iván Todorovich (Druidoo)
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _name = "account.move"
+    _inherit = ["account.move", "comment.template"]
+
+    comment_template1_ids = fields.Many2many(
+        comodel_name="base.comment.template",
+        string="Top Comment Template",
+        domain=[
+            ("model_ids.model", "=", "account.move"),
+            ("position", "=", "before_lines"),
+        ],
+    )
+    comment_template2_ids = fields.Many2many(
+        comodel_name="base.comment.template",
+        string="Bottom Comment Template",
+        domain=[
+            ("model_ids.model", "=", "account.move"),
+            ("position", "=", "after_lines"),
+        ],
+    )
```

Comment:
I think you will need to give explicit names to this relations' columns, otherwise the ORM will just generate them from the model names, and as there are 2 relations between the same modules, it will cause the conflict in https://travis-ci.com/github/OCA/account-invoice-reporting/jobs/500445740#L1133

Also, as you will already need migration scripts, why not giving a more distinct name to these, like `comment_template_top_ids` and `comment_template_bottom_ids`?